loops.js

```diff
@@ -88,6 +112,30 @@
 // FizzBuzz
 // etc.
 // ==========================================
-
-
-
+console.log('');//witregel na vorige opdracht
+
+let fizz = null;//initialiseer booleans voor de lus
+let buzz = null;
+
+for(let n = 1; n<=100 ;n++)//doe een lus voor getal n = 1 tot en met 100
+{   //test deelbaar door 3 en 5
+    fizz = n%3 ===0;
+    buzz = n%5 ===0;
+
+    if(fizz && !buzz)//deelbaar door 3, maar niet 5
+    {
+        console.log('Fizz');
+    }
+    else if(!fizz && buzz)//deelbaar door 5, maar niet 3
+    {
+        console.log('Buzz');
+    }
+    else if(fizz && buzz)//deelbaar door 3 en 5
+    {
+        console.log('FizzBuzz');
+    }
```

Comment:
```suggestion
    if(fizz && buzz)//deelbaar door 3 en 5
    {
        console.log('FizzBuzz');
    }
    else if(fizz)//deelbaar door 3, maar niet 5
    {
        console.log('Fizz');
    }
    else if(buzz)//deelbaar door 5, maar niet 3
    {
        console.log('Buzz');
    }
```

Scheelt weer wat code 😉